entities/regions.py

```diff
@@ -0,0 +1,42 @@
+from random import randint
+from random import choice
+from config import Config
+
+
+class Regions(object):
+
+    def __init__(self, region_id, subregion_id):
+        self.region_id = region_id
+        self.subregion_id = subregion_id
+
+    def region(self):
+        global region
+        region = ([(self.subregion_id) * Config.subregion_length, (self.subregion_id+1)* Config.subregion_length ],
+                               [0, Config.subregion_length])
+        return region
+    def hub(self):
+        global hub
+        hub = [(self.subregion_id+1) * Config.subregion_length - (Config.subregion_length // 2),
+               Config.subregion_length // 2]
+        return hub
+
+
+def create_regions():
```

Comment:
A file hosts either a funcion, _or_ a class ("One thing per file"). Never both. Python does not force this (most languages don't), but it's just good practice.

So this function should not be here, and be placed somewhere else.